pkl-core/src/test/files/LanguageSnippetTests/input/api/pcfRenderer1.pkl

```diff
@@ -15,21 +15,21 @@ parrot {
   new {
     // multiline string that is a property within an element
     lastName = """
-      Flinstone
+      Flintstone
```

Comment:
I understand this is a test, but I don't see any particular reason to use `Flinstone` here.